TOC-tidb-cloud.md

```diff
@@ -795,7 +795,14 @@
 - Support Plan
   - [Connected Care Overview](/tidb-cloud/connected-care-overview.md)
   - [Connected Care Details](/tidb-cloud/connected-care-detail.md)
-  - [Get Support](/tidb-cloud/tidb-cloud-support.md)
+  - Introduction to TiDB Cloud Connected Care Support Service Features
+    - [Connected: AI chat in IM](/tidb-cloud/connected-ai-chat-in-im.md)
+    - Connected: IM ticket creation and update subscription
+      - [Slack ticket creation and update subscription](/tidb-cloud/connected-slack-ticket-creation.md)
+      - [Lark ticket creation and update subscription](/tidb-cloud/connected-lark-ticket-creation.md)
+    - Connected: IM interaction for support tickets
+      - [Slack interaction for support tickets](/tidb-cloud/connected-slack-ticket-interaction.md)
+      - [Lark interaction for support tickets](/tidb-cloud/connected-lark-ticket-interaction.md)
```

Comment:
Please add the links here for both Clinic, and Alert as well.
By the way - in both pages (Clinic and Alerts) - add illustration pictures.